tools/clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -15357,10 +15464,14 @@ void DiagnoseEntry(Sema &S, FunctionDecl *FD) {
   case DXIL::ShaderKind::ClosestHit: {
     return DiagnoseClosestHitEntry(S, FD, Attr);
   }
-  case DXIL::ShaderKind::Compute:
+  // TODO: Create a Compute function, move compute only diags over.
```

Comment:
Now that you created a `DiagnoseComputeEntry` function, what is this TODO needed for?